src/db.rs

```diff
@@ -1969,9 +1957,13 @@ mod tests {
         )
         .await
         .unwrap();
-        db.delete_with_options(&[b'b'; 32], &write_options)
+        db.delete_with_options(&[b'b'; 31], &write_options)
```

Comment:
Curious why this changes?

Reply:
i remember that this test case goes into a hang because of memtable's size changed, as it'll keep awaiting durable to be finished, but the flusher will not flush wal/memtable before reaching the size. changing this size can help pass this test 🥹.

Reply:
Why did the memtable size change? Is it 'cuz of the new `estimated_size` function in `RowEntry`? Was removed from the calculation that used to be included?

Reply:
before this change, the MemTable did not include RowEntry as the value, and the key was not KvTableInternalKey, so the size might become bigger after this change 😲